test/controllers/passwords_controller_test.rb

```diff
@@ -29,4 +29,24 @@ def put_update_with_params
     put_update_with_params
     assert_redirected_to custom_path
   end
+
+  test 'redirect to http://test.host/users/sign_in after successfully sent reset password instructions' do
+    Devise::PasswordsController.any_instance.stubs(:successfully_sent?).with(@user).returns(true)
+
+    post :create, params: { "user" => { "email" => @user.email } }
+    assert_redirected_to 'http://test.host/users/sign_in'
+  end
+
+  test 'redirect to http://test.host/users/password/new after unsuccessfully sent reset password instructions' do
+    Devise::PasswordsController.any_instance.stubs(:successfully_sent?).with(@user).returns(false)
+
+    post :create, params: { "user" => { "email" => @user.email } }
+    assert_redirected_to 'http://test.host/users/password/new'
+  end
+
+  test 'nothing raised after unsuccessfully sent reset password instructions for users that exist' do
```

Comment:
This test case exercises the same code as the previous test, and `assert_nothing_raised` is not a very helpful assertion to have around since we don't expect the previous test to raise anything (and I'm planning on removing the other occurrences of this assertion from our test suite).

Can you please remove this test case?